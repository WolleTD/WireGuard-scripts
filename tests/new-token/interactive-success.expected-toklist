[A-Z2-7]{8}		172\.16\.1\.1/32		peer1
