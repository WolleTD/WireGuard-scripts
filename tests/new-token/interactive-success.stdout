Add wg-setup tokens to /tmp/tmp\..+/infile
Name: AllowedIPs: 
[A-Z2-7]{8}		172\.16\.1\.1/32		peer1
Add another token\? \[y/N\] 
